dandi/dandiapi.py

```diff
@@ -474,6 +474,7 @@ def dandi_authenticate(self) -> None:
         # Shortcut for advanced folks
         api_key = os.environ.get("DANDI_API_KEY", None)
         if api_key:
+            lgr.debug("Using api key from DANDI_API_KEY env variable")
```

Comment:
```suggestion
            lgr.debug("Using API key from DANDI_API_KEY environment variable")
```

Reply:
will do, thanks

Reply:
@yarikoptic You didn't commit my suggested change.